src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -182,6 +182,23 @@ object Helpers extends SparkSessionWrapper {
     fs.exists(path)
   }
 
+  /**
+   * Check whether a path with a regex pattern exists
+   *
+   * @param name file/directory name as a regex
+   * @return
+   */
+  def pathPatternExists(name: String): Boolean = {
+    val path = new Path(name)
+    val fs = path.getFileSystem(spark.sparkContext.hadoopConfiguration)
+    val globPath = fs.globStatus(path)
+    if (globPath.isEmpty)
+      false
+    else
+       true
+  }
```

Comment:
This is still very strange to me -- I think it's what we discussed but please double think just to make sure .

Also -- please make this private[overwatch] -- no user should be using this.